### ISSUE TITLE
spanish support for sumy

### ISSUE THREAD

**Issue by stshoot** [miso-belica/sumy#43]
Hi, I would like that you add spanish support for the project.
In my town there are a research group very interested in this project with the spanish support

**Comment by miso-belica** [comment]
Hi,
There are only few language dependend parts:
- `Tokenizer` is just [thin wrapper](https://github.com/miso-belica/sumy/blob/dev/sumy/nlp/tokenizers.py#L40) around NLTK so it supports spanish.
- `Stemmer` is just [thin wrapper](https://github.com/miso-belica/sumy/blob/dev/sumy/nlp/stemmers/__init__.py#L26) around NLTK so it supports spanish.
- Summarizers works also without stop-words or you can provide your own.

So in conclusion Spanish is supported quite well, what is missing for you?

**Comment by stshoot** [comment]
We add the spanish stopwords en data folder, but it seems that does not recognize
I think we just need a little guidance to implement it.

**Comment by miso-belica** [comment]
@stshoot that are good news. Just [create PR](https://help.github.com/articles/creating-a-pull-request/) and I'll look at it :)

**Comment by stshoot** [comment]
This is a test code with edmundson

In data/stopwords I add the spanish.txt

```python

from __future__ import absolute_import
from __future__ import division, print_function, unicode_literals

from sumy.parsers.html import HtmlParser
from sumy.parsers.plaintext import PlaintextParser
from sumy.nlp.tokenizers import Tokenizer
from sumy.summarizers.edmundson import EdmundsonSummarizer as Summarizer
from sumy.nlp.stemmers import Stemmer
from sumy.utils import get_stop_words

LANGUAGE = "spanish"
SENTENCES_COUNT = 5


if __name__ == "__main__":
    url = "http://es.wikipedia.org/wiki/Objeto_volador_no_identificado"
    parser = HtmlParser.from_url(url, Tokenizer(LANGUAGE))
    stemmer = Stemmer(LANGUAGE)
   
    summarizer = Summarizer(stemmer)
    summarizer.stop_words = get_stop_words(LANGUAGE)

    for sentence in summarizer(parser.document, SENTENCES_COUNT):
        print(sentence)
```
 but get this: 
``` python
raise ValueError("Set of bonus words is empty. Please set attribute 'bonus_words' with collection of words.")
ValueError: Set of bonus words is empty. Please set attribute 'bonus_words' with collection of words.

```

**Comment by miso-belica** [comment]
I think error message is pretty clear, Edmundson method requires list of bonus words, so you should provide it `summarizer.bonus_words = get_bonus_words()`.

**Comment by stshoot** [comment]
Thanks for your help, It's an amazing code!!

**Comment by miso-belica** [comment]
@stshoot good to hear. I would appreciate if you attach the list of used bones/null/stigma words here so sumy can speak Spanish natively :) Thanks for nice words

**Comment by stshoot** [comment]
Of curse. And I'm interested in summarizing legal documents in spanish of my country and think "summy" is really a good choice. Am I able to contact you by email?

**Comment by miso-belica** [comment]
My e-mail is in `setup.py` file. Feel free to write me :)

**Comment by happyshivi** [comment]
Can you please provide the path for the get_bonus_words() function for this line 
summarizer.bonus_words = get_bonus_words()
